src/utilities/get-content.ts

```diff
@@ -0,0 +1,21 @@
+import { getAuthoringApiUrl } from "../authoring/utils/authoring-api";
+import { urlParams } from "./url-params";
+
+export const getContent = async (input: RequestInfo, init?: RequestInit): Promise<Response> => {
```

Comment:
This name is a little generic. `getAuthoredContent` seems more clear to me, but perhaps I'm missing someway that it is used.

Reply:
It is a pass through to fetch if we aren't in authoring preview mode.  Maybe I should call it something else?